src/Yoti.Auth/Images/Image.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Yoti.Auth.Images
+{
+    public abstract class Image
```

Comment:
Do you think this should be called `BaseImage` or `AbstractImage` so that we don't have as much of a stutter in `Yoti.Auth.Images.Image`?

Reply:
Either _Image_, _BaseImage_, _ImageBase_ or _AbstractImage_ is fine.
Maybe not _AbstractImage_ - it's a bit Picasso.
.Net has a convention of using _xxxBase_, but there's no need for you to stack to that.

Reply:
I prefer ImageBase anyway, so will change to that!

Reply:
We do have `BaseAttribute` currently though, so I'm thinking `BaseImage` would be more consistent, what do you think?

Reply:
Have changed to `Image`, as it's not necessary to convey it's a base class to clients.